sw/Modules/Comms/Inc/can_sensors.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * can_sensors.h
+ *
+ *  Created on: Mar 19, 2025
+ *      Author: Hardy Yu
+ */
+#ifndef CAN_SENSORS_H_
+#define CAN_SENSORS_H_
+
+#include <stdint.h>
+#include "canard.h"
+#include "dronecan_msgs.h"
+#include "imu.h"
+
+typedef enum {
+    COMMS_STATUS_OK = 0,
+    COMMS_STATUS_ERR = 1,
+} CAN_COMMS_STATUS_E;
+
+void
+convertImuToDroneCAN (const ImuData_S *src,
```

Comment:
Nit: bring these back onto the same line.

(Even if we have these on the same line, and we block-align the variables, we still stay under 80 characters)

<img width="1455" alt="image" src="https://github.com/user-attachments/assets/80371984-87be-4e0f-a26c-2a9a6581a62d" />